### ISSUE TITLE
UnicodeDecode error in config.py for coverage 5.0 and 5.0.1 which did not exist in 4.5.4 (bug for Windows only)

### ISSUE THREAD

**Issue by tleonhardt** [coveragepy/coveragepy#913]
**Describe the bug**
For the `cmd2` Python package, our Windows unit tests on AppVeyor began failing with a UnicodeDecodeError in coverage/config.py starting with version 5.0.  Forcing a max version less than 5.0 to get 4.5.4 returned everything to working:
https://github.com/python-cmd2/cmd2/issues/836

For coverage 5.0.1 the end of the traceback looks like the following:
```sh
 File "c:\projects\cmd2\.tox\py37\lib\site-packages\pytest_cov\plugin.py", line 186, in start
   self.cov_controller.start()
 File "c:\projects\cmd2\.tox\py37\lib\site-packages\pytest_cov\engine.py", line 179, in start
   config_file=self.cov_config)
 File "c:\projects\cmd2\.tox\py37\lib\site-packages\coverage\control.py", line 193, in __init__
   concurrency=concurrency, context=context,
 File "c:\projects\cmd2\.tox\py37\lib\site-packages\coverage\config.py", line 516, in read_coverage_config
   config_read = config.from_file(fname, our_file=our_file)
 File "c:\projects\cmd2\.tox\py37\lib\site-packages\coverage\config.py", line 327, in from_file
   self._config_contents = f.read()
 File "c:\projects\cmd2\.tox\py37\lib\encodings\cp1252.py", line 23, in decode
   return codecs.charmap_decode(input,self.errors,decoding_table)[0]
nicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 650: character maps to <undefined>
ERROR: InvocationError for command 'C:\projects\cmd2\.tox\py37\Scripts\py.test.EXE' --cov --junitxml=junit/test-results.xml (exited with code 1)
```

**To Reproduce**
How can we reproduce the problem?  Please *be specific*.
1. What version of Python are you using?
3.6 or 3.7
1. What version of coverage.py are you using? The output of `coverage debug sys` is helpful.
Works with 4.5.4.  Breaks with 5.0 or 5.0.1
1. What versions of what packages do you have installed? The output of `pip freeze` is helpful.
Other likely packages of relevance include (but these are the same version when tests pass with coverage 4.5.4 and when tests break with coverage >= 5.0):
* tox 3.14.3
* pytest-cov 2.8.1
1. What code are you running?  Give us a specific commit of a specific repo that we can check out.
[cmd2](https://github.com/python-cmd2/cmd2) master branch continuous integration (CI) unit tests runs upon Git commit and/or PR.  The problem only exists on Windows.  Linux tests on TravisCI and macOS tests on Azure Pipelines work fine regardless of the version of coverage.
1. What commands did you run?
tox -e %TOXENV%
**Expected behavior**
Unit tests to pass without a `UnicodeDecodeError`

**Additional context**
For now we put in a PR to force `coverage` version to be < 5.0, but this isn't a long term solution:
https://github.com/python-cmd2/cmd2/pull/838

For whatever it is worth, here are links to a few of our possibly relevant configuration files:
* [.coveragerc](https://github.com/python-cmd2/cmd2/blob/master/.coveragerc)
* [.appveyor.yml](https://github.com/python-cmd2/cmd2/blob/master/.appveyor.yml)
* [tox.ini](https://github.com/python-cmd2/cmd2/blob/master/tox.ini)
* [setup.py](https://github.com/python-cmd2/cmd2/blob/master/setup.py)

Thanks for maintaining an extremely useful module!

If you believe the problem is on our end, then we would appreciate any advice you might have on how to fix it.  Thanks ;-)

**Comment by geekvikas** [comment]
Happens in Ubuntu docker image too but slight different error message is thrown.. 

Couldn't use data file '/home/circleci/project/test_results/.coverage': database disk image is malformed

**Comment by nedbat** [comment]
It looks like this line in your .coveragerc is the issue:
```
# (string, default “htmlcov”): where to write the HTML report files.
```
The quotes are non-ASCII curly quotes.  I don't know why 5.x fails to read them when 4.5 did.  You can work around the issue for now by replacing those characters with ASCII quotes.

**Comment by tleonhardt** [comment]
@nedbat Ned, thanks a bunch for the quick response and figuring out how to fix our .coveragerc file!  Your proposed solution worked great: https://github.com/python-cmd2/cmd2/pull/842

From our end we are fine and you can close this issue if you like.  I'll leave it open in case you want to figure out why 5.x fails to read that when 4.5 did.  I didn't think weird quotes in a comment would be a problem.

**Comment by nedbat** [comment]
I'd like to fix coverage.py to not fail in this way, so I'll leave the issue open. Glad your repo is sorted out.

**Comment by nedbat** [comment]
This was fixed in version 5.2 of coverage.py.
